### ISSUE TITLE
[Doc] The logo file in the lower left corner of the page is 404

### ISSUE THREAD

**Issue by Createsequence** [apache/eventmesh#4996]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/eventmesh/issues?q=is%3Aissue) and found no similar issues.

### Documentation Related

I found that the logo file in the lower left corner of the page is not loaded correctly：

<img width="1491" alt="image" src="https://github.com/apache/eventmesh/assets/49221670/b6e4c1e6-9f60-4767-801e-be4cf39b2af0">

After opening the browser console, I found that the file is 404：

<img width="507" alt="image" src="https://github.com/apache/eventmesh/assets/49221670/d0f68f7d-26ab-411b-9957-c1bb0bec492e">

Was this file renamed or moved to another directory? I thought I might be able to try to fix this. :)

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct) *

**Comment by Pil0tXia** [comment]
Quite detailed observation. You can find the website code at https://github.com/apache/eventmesh-dashboard repo.

The problem is because the logo file uses a relative path reference. Just replace it with an absolute path.

**Comment by Createsequence** [comment]
> [eventmesh-dashboard](https://github.com/apache/eventmesh-dashboard)

Sorry for the late reply. I don't seem to find the develop branch in the eventmesh-dashboard, and I can't find the /docs directory in the main branch. Which branch should I deal with this problem?

**Comment by Createsequence** [comment]
> Quite detailed observation. You can find the website code at https://github.com/apache/eventmesh-dashboard repo.
> 
> The problem is because the logo file uses a relative path reference. Just replace it with an absolute path.

Sorry for the late reply. I don't seem to find the develop branch in the eventmesh-dashboard, and I can't find the /docs directory in the main branch. Which branch should I deal with this problem?

**Comment by Pil0tXia** [comment]
Correct, the logo file is stored in https://github.com/apache/eventmesh-site repo.

**Comment by Createsequence** [comment]
@qqeasonchen ，@Pil0tXia Thanks for the guidance, I think I fixed the problem.

But should I close the issue and reopen it in the https://github.com/apache/eventmesh-site repo before submitting the PR?
